.github/workflows/main.yml

```diff
@@ -142,6 +142,25 @@ jobs:
     - name: run tests
       run: PYTEST_BACKENDS=$BACKENDS PYTEST_EXPRESSION="not udf and not hdfs" ./ci/run_tests.sh
 
+  Tests_spark:
+    name: Tests Spark
+    runs-on: ubuntu-latest
+    env:
+      BACKENDS: "spark pyspark"
+    strategy:
+      fail-fast: false
+      matrix:
+        python_version: ["3.7"]
```

Comment:
Are we testing python 3.8 in other pipelines?

Reply:
All pipelines test on both 3.7 and 3.8 except the one for OmniSciDB. Let me try enabling 3.8 for this